Config/Processor/BuilderProcessor.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Overblog\GraphQLBundle\Config\Processor;
+
+use Overblog\GraphQLBundle\Definition\Builder\MappingInterface;
+use Overblog\GraphQLBundle\Relay\Connection\BackwardConnectionArgsDefinition;
+use Overblog\GraphQLBundle\Relay\Connection\ConnectionArgsDefinition;
+use Overblog\GraphQLBundle\Relay\Connection\ForwardConnectionArgsDefinition;
+use Overblog\GraphQLBundle\Relay\Mutation\MutationFieldDefinition;
+use Overblog\GraphQLBundle\Relay\Node\GlobalIdFieldDefinition;
+use Overblog\GraphQLBundle\Relay\Node\NodeFieldDefinition;
+use Overblog\GraphQLBundle\Relay\Node\PluralIdentifyingRootFieldDefinition;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+final class BuilderProcessor implements ProcessorInterface
+{
+    const BUILDER_FIELD_TYPE = 'field';
+    const BUILDER_ARGS_TYPE = 'args';
+
+    /** @var MappingInterface[] */
+    private static $argsBuilderClassMap = [
+        'Relay::ForwardConnection' => ForwardConnectionArgsDefinition::class,
+        'Relay::BackwardConnection' => BackwardConnectionArgsDefinition::class,
+        'Relay::Connection' => ConnectionArgsDefinition::class,
+    ];
+
+    /** @var MappingInterface[] */
+    private static $fieldBuilderClassMap = [
+        'Relay::Mutation' => MutationFieldDefinition::class,
+        'Relay::GlobalId' => GlobalIdFieldDefinition::class,
+        'Relay::Node' => NodeFieldDefinition::class,
+        'Relay::PluralIdentifyingRoot' => PluralIdentifyingRootFieldDefinition::class,
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function process(array $configs)
+    {
+        foreach ($configs as &$config) {
+            if (isset($config['config']['fields']) && is_array($config['config']['fields'])) {
+                $config['config']['fields'] = self::processFieldBuilders($config['config']['fields']);
+            }
+        }
+
+        return $configs;
+    }
+
+    public static function addArgsBuilderClass($name, $argBuilderClass)
+    {
+        self::checkBuilderClass($argBuilderClass, 'args');
+        self::$argsBuilderClassMap[$name] = $argBuilderClass;
+    }
+
+    public static function addFieldBuilderClass($name, $fieldBuilderClass)
+    {
+        self::checkBuilderClass($fieldBuilderClass, 'field');
+        self::$fieldBuilderClassMap[$name] = $fieldBuilderClass;
+    }
+
+    /**
+     * @param string $builderClass
+     * @param string $type
+     */
+    private static function checkBuilderClass($builderClass, $type)
```

Comment:
Would it be good to test all those exceptions?

Reply:
I'll add some Unit tests to cover this :+1:

Reply:
done